doc/modules/classes.rst

```diff
@@ -158,8 +158,8 @@ Classes
 
    covariance.EmpiricalCovariance
    covariance.EllipticEnvelope
-   covariance.GraphLasso
```

Comment:
These should appear under deprecated at the end of the page.

Reply:
I placed it under `To be removed in 0.22`. Is it ok?